client/signup/config/flows-pure.js

```diff
@@ -344,11 +344,11 @@ export function generateFlows( {
 		// when switching from the onboarding flow.
 		{
 			name: 'import-onboarding',
```

Comment:
Why can't we remove this entire flow and all related code? Are we using it somewhere else?

Reply:
I've checked. It can be removed entirely. It is not used anywhere.